### ISSUE TITLE
undefined parse transform 'vararg' on OTP 17.5

### ISSUE THREAD

**Issue by wojtekmach** [ferd/rebar3_proper#24]
Hello, I have issues using this plugin on OTP 17.5, see error below. On OTP 18+ it works fine. What's weird is it does look like proper works on OTP 17 here: https://github.com/manopapad/proper/blob/master/.travis.yml#L13. Sorry if this is not the _proper_ place to report it.

here's minimal repo to reproduce it, just setting up the plugin on a blank new rebar3 lib project: https://github.com/wojtekmach/proper_bug

---

```
~/proper_bug[master]% rebar3 report compile
===> Compiling proper
make: `include/compile_flags.hrl' is up to date.
===> Compiling _build/default/plugins/proper/src/proper_gen.erl failed
_build/default/plugins/proper/src/proper_gen.erl:none: undefined parse transform 'vararg'

===> Plugin rebar3_proper not available. It will not be used.
Rebar3 report
 version 3.5.0
 generated at 2018-03-04T20:33:48+00:00
=================
Please submit this along with your issue at https://github.com/erlang/rebar3/issues (and feel free to edit out private information, if any)
-----------------
Task: compile
Entered as:
  compile
-----------------
Operating System: x86_64-apple-darwin16.7.0
ERTS: Erlang/OTP 17 [erts-6.4] [source] [64-bit] [smp:8:8] [async-threads:0] [hipe] [kernel-poll:false]
Root Directory: /Users/wojtek/.asdf/installs/erlang/17.5
Library directory: /Users/wojtek/.asdf/installs/erlang/17.5/lib
-----------------
Loaded Applications:
bbmustache: 1.3.0
certifi: 2.0.0
cf: 0.2.2
common_test: 1.10
compiler: 5.0.4
crypto: 3.5
cth_readable: 1.3.2
dialyzer: 2.7.4
edoc: 0.7.16
erlware_commons: 1.0.4
eunit: 2.2.9
eunit_formatters: 0.5.0
getopt: 1.0.1
inets: 5.10.6
kernel: 3.2
providers: 1.7.0
public_key: 0.23
relx: 3.24.3
sasl: 2.4.1
snmp: 5.1.1
ssl_verify_fun: 1.1.3
stdlib: 2.4
syntax_tools: 1.6.18
tools: 2.7.2

-----------------
Escript path: /Users/wojtek/bin/rebar3
Providers:
  app_discovery as clean compile compile cover ct deps dialyzer do edoc escriptize eunit get-deps help install install_deps list lock new path pkgs release relup report shell state tar tree unlock update upgrade upgrade upgrade version xref
```

**Comment by ferd** [comment]
Well that's odd, especially if it works on OTP-17 and not 17.5. Their own CI seems to use 17.4 as well. I don't know that we can do much to fix it within the plugin though. If PropEr + rebar3 build well together in all versions but 17.5 maybe there's something odd with that version specifically.

**Comment by wojtekmach** [comment]
I just tried it on both 17.0 and 17.5 and get the same result in both cases.

**Comment by wojtekmach** [comment]
Good news, they have since released proper 1.3 and I don't see the problem anymore.
